omnibus/package-scripts/chef-workstation/postinst

```diff
@@ -31,6 +31,12 @@ for binary in $binaries; do
   ln -sf "$INSTALLER_DIR/bin/$binary" $PREFIX/bin || error_exit "Cannot link $binary to $PREFIX/bin"
 done
 
+# everyone needs a .chef dir so let's create it for them
+if [ ! -d ~/.chef/ ]; then
```

Comment:
It is recommended to use double quotes to avoid problems during string expansion,
`"~/.chef"` and when you do that, you can't use `~` so I would recommend the use
of `$HOME` here and in the next two lines.

Reply:
Fixed